components/HomepageNews/HomepageNews.vue

```diff
@@ -65,6 +65,22 @@ export default {
     }
   },
 
+  computed: {
+    /**
+     * Filter news to remove past events
+     * @returns {Array}
+     */
+    upcomingNews: function() {
+      const newsToFilter = [...this.news]
+      newsToFilter.forEach(event => {
+        this.isUpcoming(event)
+          ? ''
+          : newsToFilter.splice(newsToFilter.indexOf(event), 1)
+      })
+      return newsToFilter
+    }
```

Comment:
Optional, but this could be simplified to user the [filter()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter) method.
```suggestion
    upcomingNews: function() {
      return this.news.filter(event => this.isUpcoming(event))
    }
```

Reply:
Thanks for the suggestion, much more simple!